controllers/idle_handlers.go

```diff
@@ -18,16 +18,55 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"os"
 
 	ranv1alpha1 "github.com/openshift-kni/lifecycle-agent/api/v1alpha1"
+	"github.com/openshift-kni/lifecycle-agent/controllers/utils"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+func getOSName(seedImageVersion string) string {
```

Comment:
Point of this is to also use it in the prep handler.

Reply:
can you point to where in prepHandler? osname is only used in the bash scripts and it is extracted from the seed image as I said in the previous comment. If we want to change that it is out of scope of this PR.

Reply:
```
new_osname="rhcos_${up_ver}"
```
Make this an arg passed in by the prep handler?

Reply:
That value comes from the seed image
```
up_ver=$(jq -r '.status.desired.version' "${img_mnt}/clusterversion.json")
if [ -z "${up_ver}" ]; then
    fatal "Failed to identify version from seed image"
fi
```
we need to handle the case when that value is not equal to the version specified in the CR

Reply:
The current prep handler scripts are just the initial implementation. The final implementation will presumably be go and would be able to make use of such functions